includes/admin/home/class-sensei-home-guides-provider.php

```diff
@@ -0,0 +1,62 @@
+<?php
+/**
+ * File containing Sensei_Home_Guides_Provider class.
+ *
+ * @package sensei-lms
+ * @since   $$next-version$$
+ */
```

Comment:
I have been adding the `ABSPATH` check to all these files. As we talked not really sure about the benefit but still maybe you want to consider adding that for consistency.